packages/components/build/esbuild.config.ts

```diff
@@ -22,10 +23,12 @@ const rootDir = 'components';
 const getComponentData = async () => {
   const paths = await tinyGlob('components/*/package.json');
 
-  return paths.map((packageJsonPath) => {
-    const file = fs.readFileSync(packageJsonPath, 'utf-8');
-    return { content: JSON.parse(file), component: packageJsonPath.split(path.sep)[1] };
-  });
+  return paths
+    .filter((packageJsonPath) => !packageJsonPath.includes('elvis-toolbox'))
```

Comment:
Maybe we could add a comment regarding why we exclude the toolbox from the build